libs/types/src/constants/queue.constants.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ *  NOTE: Each service grouping (*-api, *-watcher) prefixes its BullMQ Redis connections with a service-specific
+ *        prefix, so these queue names, while common, will be unique per-service. Thus a queue "myMessageQueue" in
+ *        one service is distinct from a queue of the same name in another service.
+ *
+ *        Since the Gateway services are designed to be fully independent of one another, there should
+ *        be no reason to construct shared queue names across services.
+ */
+
+import { AnnouncementTypeName } from '#types/enums/announcement-type.enum';
+
+/**
+ * Name of the queue that publishes account transactions to Frequency blockchain
+ */
+export const TRANSACTION_PUBLISH_QUEUE = 'transactionPublish';
```

Comment:
I thought about it...

Technically, all services could use all the queues, and they would not cross service boundaries, because we prefix our BullMQ connection with the service name, so the actual Redis queue key would be unique to one service.

So, I think breaking this file up might be overkill, but certainly we could offset the groupings with commented sections for the different services... thoughts?

Reply:
I think maybe using namespaces to organize them might be helpful. The reason might be just an easy way to lookup which queues are used where and additional protections against wrong usage.

Reply:
I think since we are publishing and consuming it multiple times in workers it makes it risky but not a big deal.

Reply:
Hmm, I think you're right. Top-level (ie, entire module) namespaces are frowned upon nowadays in Typescript in favor of ES Module imports; but multiple sub-namespaces in a file are still appropriate. I'll update.